src/services/codefixes/inferFromUsage.ts

```diff
@@ -1101,15 +1101,16 @@ namespace ts.codefix {
                 const genericParam = genericSig.parameters[i];
                 const usageParam = usageSig.parameters[i];
                 const isRest = genericSig.declaration && isRestParameter(genericSig.declaration.parameters[i]);
-                if (!usageParam) {
+                if (!usageParam/* || !usageParam.valueDeclaration || !genericParam.valueDeclaration*/) {

```

Comment:
Mm?

Reply:
Leftover from my first attempt, same as the extraneous `!` above.